bugsnag-spring/src/main/java/com/bugsnag/BugsnagSpringConfiguration.java

```diff
@@ -48,21 +48,25 @@ ScheduledTaskBeanLocator scheduledTaskBeanLocator() {
      * messages as they effectively duplicate error reports for unhandled exceptions.
      */
     @PostConstruct
+    @SuppressWarnings("checkstyle:emptycatchblock")
     void excludeLoggers() {
-        // Exclude Tomcat logger when processing HTTP requests via a servlet.
-        // Regex specified to match the servlet variable parts of the logger name, e.g.
-        // the Spring Boot default is:
-        // [Tomcat].[localhost].[/].[dispatcherServlet]
-        // but could be something like:
-        // [Tomcat-1].[127.0.0.1].[/subdomain/].[customDispatcher]
-        BugsnagAppender.addExcludedLoggerPattern("org.apache.catalina.core.ContainerBase."
-                + "\\[Tomcat.*\\][.]\\[.*\\][.]\\[/.*\\][.]\\[.*\\]");
+        try {
+            // Exclude Tomcat logger when processing HTTP requests via a servlet.
+            // Regex specified to match the servlet variable parts of the logger name, e.g.
+            // the Spring Boot default is:
+            // [Tomcat].[localhost].[/].[dispatcherServlet]
+            // but could be something like:
+            // [Tomcat-1].[127.0.0.1].[/subdomain/].[customDispatcher]
+            BugsnagAppender.addExcludedLoggerPattern("org.apache.catalina.core.ContainerBase."
+                    + "\\[Tomcat.*\\][.]\\[.*\\][.]\\[/.*\\][.]\\[.*\\]");
 
-        // Exclude Jetty logger when processing HTTP requests via the HttpChannel
-        BugsnagAppender.addExcludedLoggerPattern("org.eclipse.jetty.server.HttpChannel");
+            // Exclude Jetty logger when processing HTTP requests via the HttpChannel
+            BugsnagAppender.addExcludedLoggerPattern("org.eclipse.jetty.server.HttpChannel");
 
-        // Exclude Undertow logger when processing HTTP requests
-        BugsnagAppender.addExcludedLoggerPattern("io.undertow.request");
+            // Exclude Undertow logger when processing HTTP requests
+            BugsnagAppender.addExcludedLoggerPattern("io.undertow.request");
+        } catch (Throwable ignored) {
+        }
```

Comment:
🚗 ))) Drive-by review: There should be a comment indicating why an exception is ignored here. Also, can the `Throwable` be a more specific subclass?

Reply:
I may be wrong, but after we do this I think we'd be able to remove the `@SuppressWarnings` annotation. I think a comment is sufficiently non-empty.